lib/autospec/qunit_runner.rb

```diff
@@ -32,7 +32,7 @@ def reloaders
     # Discourse specific
     reload(%r{^discourse/tests/javascripts/fixtures/.+_fixtures\.js(\.es6)?$})
     reload(%r{^discourse/tests/javascripts/(helpers|mixins)/.+\.js(\.es6)?$})
-    reload("app/assets/javascripts/discoruse/tests/javascripts/test_helper.js")
```

Comment:
no path by this name exists...

Reply:
I suspect the file should be `app/assets/javascripts/discourse/tests/test_helper.js`